server/controllers/userControllers.js

```diff
@@ -0,0 +1,19 @@
+const { User } = require("../models/userModel");
+
+exports.changeProfilePicture = async (req, res) => {
+  let email = req.body.email;
```

Comment:
```suggestion
  const email = req.body.email;
```

Use const whenever possible, so you differentiate what variables are being modified and what are constants. It'll be easier to debug later on as well :)